Simplenote/build.gradle

```diff
@@ -30,9 +30,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "2.9"
+            versionName "2.10-rc-2"
```

Comment:
I had already shipped 2.10-rc-1 on the 24th of August, when I started a code freeze before catching up with Slack properly and learning it should have been delayed.